bgpd/bgp_route.c

```diff
@@ -12336,6 +12336,11 @@ static const char *table_stats_strs[][2] = {
 struct bgp_table_stats {
 	struct bgp_table *table;
 	unsigned long long counts[BGP_STATS_MAX];
+
+	unsigned long long
```

Comment:
Do you think it is better to have this prefix_len_count array per address family ? or one for V4/V6 and another for EVPN ?

In EVPN case this prefix_len is bit different then a v4/v6 prefix length and is based on whether route is Type-1, Type-2 or Type-5 etc.